docs/pages/management/guides/gcp-tags.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: GCP Labels and Tags as Teleport Agent Labels
+description: How to set up Teleport agent labels based on GCP labels and tags
+h1: Sync GCP Labels/Tags and Teleport agent Labels
+---
+
+When running on an Google Compute Engine instance, Teleport will automatically detect and import GCP labels and tags as
+Teleport labels for SSH nodes, applications, databases, and Kubernetes clusters. Both tags and labels imported
```

Comment:
Odd that we implemented this for everything but desktops - any particular reason?

Reply:
It might have been because the imported labels would apply to the desktop service itself and not the windows desktops it manages, but I'm not sure. It would be easy to add if we wanted to.